### DISCUSSION TITLE
cli: add step back feature (redux)

### DISCUSSION THREAD

**Discussion by hydrargyrum** [asciinema/discussions#76] [Ideas]
I am conscious this feature has been requested multiple times but everytime it was closed without being addressed properly (for example asciinema/asciinema#278, asciinema/asciinema#133).

I understand stepping back is not as efficient as stepping forward, but couldn't it simply be implemented by replaying the beginning of the file until the desired frame, without timings (like `asciinema cat`)?

For example, `head -50 /path/to/asciinema.recording | asciinema cat -`, plays the first 49 frames of the recording. Seeking to previous frame would be as simple as running (the equivalent of) `head -49 /path/to/asciinema.recording | asciinema cat -`. Seeking to a specific time can be done the same way because we have timing info for each frame.

**Comment by hydrargyrum** [comment]
I just realized that feature **is** in the web player, but not in the CLI player. Why not implementing it the same way?

**Comment by cizra** [comment]
I missed this feature (along with seeking with keyboard) in `asciinema play`, so I had GPT-5.5 slop up a draft PR.
See https://github.com/asciinema/asciinema/pull/744  - I don't speak Rust myself, so I'm not sure how good it is. It certainly works in my machine :)
